src/Personalization.cs

```diff
@@ -0,0 +1,63 @@
+using Newtonsoft.Json;
+using Stream.Models;
+using Stream.Utils;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Stream
+{
+    public class Personalization : IPersonalization
+    {
+        private readonly StreamClient _client;
+
+        internal Personalization(StreamClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<IDictionary<string, object>> GetAsync(string endpoint, IDictionary<string, object> data)
+        {
+            var request = _client.BuildPersonalizationRequest(endpoint + "/", HttpMethod.Get);
+            foreach (KeyValuePair<string, object> entry in data)
+            {
+                request.AddQueryParameter(entry.Key, Convert.ToString(entry.Value));
+            }
+
+            var response = await _client.MakeRequestAsync(request);
+            if ((int)response.StatusCode < 300)

```

Comment:
why is the response validation logic different in this client? It's best to be unified across all files so it should be `response.StatusCode == HttpStatusCode.OK` + likewise for all other methods

Reply:
Agreed, I didn't touch it btw, I overlooked it